.circleci/config.yml

```diff
@@ -967,41 +967,43 @@ jobs:
 
   coverage:
     # https://circleci.com/developer/images?imageType=machine
-    machine:
-      image: ubuntu-2004:202201-02
+    docker:
+      - image: rust:1.72.0
```

Comment:
The machone related comment can be removed too

Reply:
I was referring to `# https://circleci.com/developer/images?imageType=machine` which is still in here

Reply:
Somehow I completely missed that one